plugins/utilities/sandbox.py

```diff
@@ -0,0 +1,6034 @@
+
+        for bot in bot_list:
+            if bot not in lmao_bots:
+                lmao_bots.append(bot)
+            i = lmao_bots.index(bot)
+            if not LAH[i]:
+                player_pts.remove(host)
+            if not LAP[i]:
+                player_pts = [k for k in player_pts if k not in not_host]
+            if not LAB[i]:
+                player_pts = [k for k in player_pts if k not in bad_bots]
+            if not LAF[i]:
+                player_pts = [k for k in player_pts if k not in good_bots]
+            bot.set_player_points(player_pts)
+            bot.update_ai()
+
+        for bot in self.get_living_bots():
+            if bot not in lmao_bots2 and bot not in lmao_bots:
+                lmao_bots2.append(bot)
+
+    def setup_custom(self, spaz) -> None:
+        spaz.source_player = self.source_player
+        self.set_custom_text(spaz)
+        for i in Nice.pending:
+            if i == "sp":
+                Nice.give_sp(Nice, spaz)
+            if i == "speed":
+                spaz.node.hockey = True
+            if i == "constant_heal":
+                Nice.constant_heal(Nice, spaz)
+            if i == "toxic":
+                Nice.toxic_bots.append(spaz)
+                Nice.make_toxic(Nice, spaz)
+            if i == "constant_jump":
+                spaz.on_jump_press = Nice.spaz_bot_fly(Nice, spaz.on_jump_press)
+                Nice.constant_jump(Nice, spaz)
+            if i == "infinite_curse":
+                if spaz._cursed:
+                    spaz.handlemessage(bs.PowerupMessage('health'))
+                spaz.curse()
+                Nice.spam_curse(Nice, spaz)
+
+    def set_custom_text(self, spaz) -> None:  # FLOAT
+        global nice_custom_text, nice_custom_color, testa
+        try:
+            m = bs.newnode('math',
+                           owner=spaz.node,
+                           attrs={'input1': (0, 1.2, 0),
+                                  'operation': 'add'})
+            spaz.node.connectattr('position', m, 'input2')
+            test = spaz._custom_text = bs.newnode(
+                'text',
+                owner=spaz.node,
+                attrs={'text': nice_custom_text,
+                       'in_world': True,
+                       'shadow': 1.0,
+                       'flatness': 1.0,
+                       'color': nice_custom_color,
+                       'scale': 0.0,
+                       'h_align': 'center'})
+            m.connectattr('output', spaz._custom_text, 'position')
+            bs.animate(spaz._custom_text, 'scale', {0: 0.0, 0.5: 0.01})
+            testa.append(test)
+        except:
+            pass
+
+# ba_meta require api 8
+# BroBordd touch grass
+# Copyright 2024, solely by BroBordd. All rights reserved.
+# ba_meta export plugin
+
+
+class byBordd(ba.Plugin):
+    def __init__(s):
+        mm.MainMenuWindow._refresh_in_game = Nice.Button(mm.MainMenuWindow._refresh_in_game)
+
+
+# All Textures (generated)
+all_texture = [
```

Comment:
u can get the name of textures during runtime so if a new texture gets added it wont be needed to update here

Reply:
Indeed, when I know which directory does contain the game at first

Reply:
updated, check last commit below